### ISSUE TITLE
Add a PHP coding style checker/fixer command

### ISSUE THREAD

**Issue by mlocati** [concrete5/console#9]
Referring to https://github.com/concrete5/concrete5/pull/9852#issuecomment-925275787: what about moving the c5:phpcs CLI command (and all the related code) from the core to this project?

**Comment by KorvinSzanto** [comment]
Is this command something that people would use outside of core development?

**Comment by aembler** [comment]
I would expect this to be valuable for anyone developing Concrete code, whether it's for the core or for their own custom site/package

**Comment by mlocati** [comment]
Yep, it's for developers only.
Having a coding style checker/fixer in the core or in this CLI tool would both work for me, but I'd prefer this CLI tool because:

- it can be updated more frequently than the core
- we don't have in the core some code that's only used by a 0.1% of users

**Comment by KorvinSzanto** [comment]
Fair enough, makes sense to me as long as it's useful outside of core development :+1:
